### ISSUE TITLE
The Charite association count is probably not quite right

### ISSUE THREAD

**Issue by Colossus** [ajratner/dd-genomics#200]
I can't imagine Charite doesn't contain BRCA1 and breast cancer, such as in 

BRCA1   HP:0003002  Breast carcinoma    "The _ BRCA1 _ 5382insC mutation -LRB- more recently described as c. 5266dup -RRB- is the second most frequent mutation described in this gene worldwide , according to the Breast Cancer Information Core -LRB- BIC , ."

**Comment by amwenger** [comment]
Based on my understanding, the Charite gene<->phenotype annotation set should have an entry for BRCA1<->breast cancer.  As I understand it, the Charite `gene<->phenotype` annotation set is a join of the OMIM `gene<->disease` set with the OMIM `disease<->phenotype` set.
1. `gene<->disease`: OMIM gene [113705/BRCA1](http://www.omim.org/entry/113705) is connected to the OMIM disease [604370/familial breast-ovarian cancer 1](http://www.omim.org/entry/604370).
2. `disease<->phenotype`: The [OMIM Clinical Synopsis of the disease 604370/familial breast-ovarian cancer 1](http://www.omim.org/clinicalSynopsis/604370) lists Breast Cancer / HPO:0003002.  Click "Display Options > Show Clinical IDs" to see the HPO IDs.

**Comment by Colossus** [comment]
OK, thanks! Then it's definitely a bug. Let's kill it ...

**Comment by Colossus** [comment]
BTW it's not that we don't pick up _any_ charite association. Just that a few of the ones highlighted as "not in charite" seem to be obviously wrong.

**Comment by ajratner** [comment]
@Colossus  Could our entity assignments be wrong, ie did we have the write endembl and hpo ids in those examples?

**Comment by Colossus** [comment]
Hmm, maybe that's the issue. ... Not too sure though ... I'd actually rather think not, unless the ensembl_genes table is wrong. The HP for breast cancer is definitely right ...

**Comment by ajratner** [comment]
Yeah I'd assume not but just hoping for easy fixes as always :)

**Comment by amwenger** [comment]
@Colossus I do not think this is your bug.  BRCA1 is not annotated with Breast cancer in the Charite HPO database.  It seems to be their bug, not yours.

I see in your build script that you use a hardcoded Charite build that Harendra provided.  You should upgrade to the newest build, which you can always obtain at http://compbio.charite.de/hudson/job/hpo.annotations.monthly/lastStableBuild/ALL_SOURCES_ALL_FREQUENCIES_phenotype_to_genes.txt.

**Comment by Colossus** [comment]
Aaron, thanks for commenting again, I misinterpreted your earlier comments as "yeah it's in charite and this is our bug". But what's going on there? I don't get it. Do you think they made an actual effort to clean their dataset of stuff that should be known to every moderately informed person? This is more than strange.

**Comment by amwenger** [comment]
I can not imagine they would have removed well known facts from the dataset.  And I do not know why the BRCA1<->breast cancer annotation is missing.  As I explained above, my understanding of the Charite build process is that it's a join of the OMIM gene<->disease and disease<->phenotype on disease.  Perhaps it's worth doing the join ourselves.  We know it would add at least one fact.
